index.html

```diff
@@ -4,6 +4,12 @@
     <head>
         <title>Is there a game night this week?</title>
         <link type="text/css" rel="stylesheet" href="/resources/main.css" />
+        <meta property="og:title" content="Is there a game night this week?" />
+        {% if status == 'Yes' %}
+            <meta property="og:description" content="*Yes.* {{ where }} {{ when }} {{ notes }}" />
```

Comment:
Not sure what happens if 'notes' is undef - might get the string 'undef' here, or perhaps an exception. Not sure when we ever had an invite without notes though.